### PR TITLE
Cover and fix getText re divs-with-attribtues

### PR DESCRIPTION
**Pull request by islemaster**
Baker reported losing newlines using `getText()` on an App Lab textarea after pasting text into it.  You can repro the issue [using this app](https://studio.code.org/projects/applab/pIuCJNqnxeNER7EojV4nQg).

My first step with an irregular behavior like this is to check what's different in the DOM.

**Source area**
screenshot from 2016-04-11 10-33-34

**Destination area after copy-paste**
screenshot from 2016-04-11 10-33-51

There are two notable differences in this markup:
- The first line is wrapped in a `<span>`
- The subsequent `<div>`-wrapped lines have an extra line-height style attribute.

So I wrote little tests to cover both cases.  It turns out the attribute is the problem - my regex that replaces `<div>`s with newlines was looking for exactly `"<div>"` and skipping tags with attributes.  Easy fix!

### CODE REVIEW

File: apps/test/utilityTests.js [hunk 1]
```diff
@@ -600,6 +600,28 @@ describe('utils.unescapeText', function () {
...
+
+  it('Does not add newline for span-wrapped leading line', function () {
+    var input, expected;
+
+    input = '<span>Line1</span><div>Line2</div>';
+    expected = [
+      'Line1',
+      'Line2'
+    ].join('\n');
+    assert.equal(expected, unescapeText(input), 'first line span');
```
Comment (Bjvanminnen) [inline apps/test/utilityTests.js:622]: Test description says "does not add newline", but then we validate that we do add line breaks? Having two lines does seem like the behavior I would expect here fwiw

**Comment by Bjvanminnen** [comment]
One clarification on test description, but otherwise lgtm. Nice job adding tests to cover these edge cases :)

File: apps/test/utilityTests.js (hunk 1 above)
Comment (islemaster) [inline apps/test/utilityTests.js:622]: Hm - should rename?  The point is that `<span>Line1</span><div>Line2</div>` maps to `Line1\nLine2` (with no _leading_ newline), not `\nLine1\nLine2` (_with_ leading newline).  We add a newline for the `<div>` but not for the `<span>`.

Comment (Bjvanminnen) [inline apps/test/utilityTests.js:622]: Ahh, maybe something like `Adds new line for div, but not span`?

**Pull request merged**